src/java.base/share/classes/sun/nio/ch/Poller.java

```diff
@@ -128,7 +128,7 @@ private void poll1(int fdVal, long nanos, BooleanSupplier supplier) throws IOExc
      * queues the file descriptor to the update thread, then parks until the file
      * descriptor is polled.
      */
-    private void poll2(int fdVal, long nanos, BooleanSupplier supplier) {
+    private void pollAsync(int fdVal, long nanos, BooleanSupplier supplier) {
```

Comment:
I don't object to renaming these private methods but "pollAsync" is confusing as the method is not asynchronously, both poll1 and poll2 are synchronous. This seems to be a drive-by change, maybe drop it from this PR as it's nothing to do with the port to AIX.

Reply:
It's true that the change is not related to the implementation. But, I felt it was of benefit to change the poll1 and poll2 method names to be more descriptive. Since the changes already refactor the Poller implementation, I felt it was a good time to do it.

I'd be happy to change pollAsync to something that was more clear. How do you feel about `pollIndirect`?

Reply:
Changing to pollIndirect is okay if you really want to change these methods.